e2e/cypress/support/commands.ts

```diff
@@ -119,3 +119,47 @@ function LoginHelper(username: string) {
     cy.wait('@getAuth');
   });
 }
+
+Cypress.Commands.add('cleanupPoliciesByIDPrefix', (idToken: string, idPrefix: string) => {
+  cy.request({
+    auth: { bearer: idToken },
+    method: 'GET',
+    url: '/apis/iam/v2beta/policies',
+    failOnStatusCode: false
+  }).then((resp) => {
+    const body = resp.body;
+    for (const policy of body.policies) {
```

Comment:
For-loops are so golang, let's do something fancier 😉 
```typescript
resp.body.find(pol => pol.id.startsWith(idPrefix))
  .forEach(pol => cy.request({
    auth: { bearer: idToken },
    method: 'DELETE',
    url: `/apis/iam/v2beta/policies/${policy.id}`,
    failOnStatusCode: false
  }));
```
⚠️ untested, probably wrong 😄

Reply:
This pattern is in a few places so I'll fix in my other PR after I rebase.